src/develop/blend_gui.c

```diff
@@ -54,14 +55,79 @@ typedef enum _iop_gui_blendif_channel_t
   ch_max = 4
 } _iop_gui_blendif_channel_t;
 
+const dt_develop_name_value_t dt_develop_blend_mode_names[] = {
```

Comment:
current dt style is to have "= {" below, can you fix this here and below for consistency.

Reply:
I was trying to indicate a difference between null terminated and un-terminated lists ;-)

For the null terminated lists of modes, all elements can be freely moved around, and not having any "special" rows that have = { at the beginning would make the changelog cleaner. But there's really no useful purpose for this, so reverted everything to standard style as requested. Thanks!